desktop_version.css

```diff
@@ -1,39 +1,20 @@
-@media only screen and (max-width: 1100px) {
-  .header-background {
-    background-repeat: no-repeat;
-    background-size: contain;
-    background-image: url("resources/Header\ bg.svg");
-    background-attachment: scroll;
-    max-width: 100%;
-    height: 721px;
-  }
-}
-
-@media only screen and (min-width: 1100px) {
-  .header-background {
-    background-image: url("resources/Header\ bg.svg");
-    background-repeat: no-repeat;
-    background-size: cover;
-    background-attachment: scroll;
-    max-width: 100%;
-    height: 721px;
-  }
-}
-
 @media only screen and (min-width: 576px) {
```

Comment:
As requested in the requirements, please adjust your media query to 768px for the best responsiveness on all mobile devices. 😀
![image](https://user-images.githubusercontent.com/42799579/192802639-0b600abc-3b07-4fd3-8b83-8fccd304efb1.png)

Reply:
Fixed.